app/models/organization_wizzard.rb

```diff
@@ -0,0 +1,31 @@
+class OrganizationWizzard
+  include Rails.application.routes.url_helpers
+
+  def initialize(organization)
+    @organization = organization
+  end
+
+  def ready?
+    have_account? && have_categories?
+  end
+
+  def not_ready?
```

Comment:
@shir in my first version I use `unless ready?` in `if current_member.owner_or_admin? && organization_wizzard.not_ready?` and `owner_or_admin?` was `!owner_or_admin?`, but we know that is not the ruby way and I solved this problem so. And when this construction was with `unless` I made a mistake. So I think, this code became more clear and more simple for peoples.